PreMailer.Net/PreMailer.Net/PreMailer.cs

```diff
@@ -129,7 +119,7 @@ public InlineResult MoveCssInline(bool removeStyleElements = false, string ignor
 				}
 			}
 
-			var html = _document.ToHtml(new AutoSelectedMarkupFormatter(_document.Doctype));
+			var html = _document.ToHtml();

```

Comment:
Are there breaking changes in AngleSharp update here?

In #169, you can pass an `IHtmlDocument` instance to PreMailer, and before we used `DocType` as formatting when producing HTML.

Can we maintain that, or isn't formatting options supported now?

Reply:
`AutoSelectedMarkupFormatter` doesn't exist in 0.11.0.